doc/TROUBLESHOOTING.md

```diff
@@ -38,17 +38,20 @@ apt install libvirt-dev
 
 ## Selinux on Debian distros
 
-* On Debian distros SELinux is enforced by qemu even if it is disabled globally, this might cause unexpected `Could not open '/var/lib/libvirt/images/<FILE_NAME>': Permission denied` errors. Double check that `security_driver = "none"` is uncommented in `/etc/libvirt/qemu.conf` and issue `sudo systemctl restart libvirt-bin` to restart the daemon.
+* On Debian distros SELinux is enforced by qemu even if it is disabled globally, this might cause unexpected `Could not open '/var/lib/libvirt/images/<FILE_NAME>': Permission denied` errors. 
+
+Double check that `security_driver = "none"` is uncommented in `/etc/libvirt/qemu.conf` and issue `sudo systemctl restart libvirt-bin` to restart the daemon.
 
 
 ## Cannot get terraform (building from src)
```

Comment:
IMHO the title is misleading. The issue here is that it cannot find the "context" package.

Reply:
Ok thx!